docs/tutorial/using-selenium-and-webdriver.md

```diff
@@ -25,27 +25,25 @@ var assert = require('assert')
 
 var app = new Application({
   path: '/Applications/MyApp.app/Contents/MacOS/MyApp'
-})
-
-app.start().then(function () {
-  // Check if the window is visible
-  return app.browserWindow.isVisible()
-}).then(function (isVisible) {
-  // Verify the window is visible
-  assert.strictEqual(isVisible, true)
-}).then(function () {
-  // Get the window's title
-  return app.client.getTitle()
-}).then(function (title) {
-  // Verify the window's title
-  assert.strictEqual(title, 'My App')
-}).catch(function (error) {
-  // Log any failures
-  console.error('Test failed', error.message)
-}).then(function () {
+});
+
+(async () => {
```

Comment:
I wonder if this example would be a little more clear if we made this a named async function instead of an IIFE. Thoughts? 💭

Reply:
Sure that would be better. Will update.